### PR TITLE
Modify Photo Model

### PR DESCRIPTION
**Pull request by elliottlin**
Add
originURL
thumbURL
respective to origin scale and resized photo URL

Remove
Urls

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,5 +103,5 @@ query {
 ### GraphQL query by curl
 
 ```
-curl -X POST -H 'Content-Type: application/json' -d '{"query": "{ photos { id,contributor,urls } }"}' localhost:9527/graphql
+curl -X POST -H 'Content-Type: application/json' -d '{"query": "{ photos { id,contributor,urls,originURL,thumbURL } }"}' localhost:9527/graphql
```
Comment (hsukvn) [inline README.md]: urls 沒刪

Comment (elliottlin) [reply README.md]: OK

File: controller/upload.go [hunk 2]
```diff
@@ -76,11 +76,10 @@ func (ctr *UploadController) PostController(c *gin.Context) {
 	photo := db.PhotoModel{
 		ID:          id,
 		Contributor: contributor,
-		Urls: []string{
-			ctr.Url + ctr.PhotoRouter + "/" + originPath + "/" + photoFileName,
-		},
-		Time:   time.Now(),
-		Masked: false,
+		OriginURL:   filepath.Join(ctr.Url, ctr.PhotoRouter, originPath, photoFileName),
```
[the remark this answers is not in the document]
Comment (elliottlin) [reply controller/upload.go]: 已修

**Comment by hsukvn** [comment]
Nice!

**Pull request merged**